src/components/toast/toast.component.js

```diff
@@ -130,6 +138,8 @@ const Toast = React.forwardRef(
             {...tagComponent(restProps["data-component"] || "toast", restProps)}
             {...toastProps}
             ref={toastContentNodeRef}
+            tabIndex={tabIndex}
+            onBlur={() => setTabIndex(undefined)}
```

Comment:
not a big deal but I'd probably prefer to check `disableAutoFocus` and only set the tab index when that's `false`. I don't mind leaving it unconditional though given that when it's true the tabindex should already be undefined.

Reply:
Yeah as far as I'm aware it will not set it if it is already undefined but I'll update anyway as there's no need for either prop being passed if `disableAutoFocus` is true